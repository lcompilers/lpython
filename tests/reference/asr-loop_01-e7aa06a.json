{
    "basename": "asr-loop_01-e7aa06a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/loop_01.py",
    "infile_hash": "fd8dbfec9ebfe2be47e713f169a8bba58de81c0d1b8988dcc86523b2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-loop_01-e7aa06a.stderr",
    "stderr_hash": "256fe74af3480ca97b87c1b6f770cee10753e5f097d5a2d79135e736",
    "returncode": 2
}