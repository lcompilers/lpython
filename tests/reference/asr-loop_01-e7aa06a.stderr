semantic error: Type mismatch in loop start and loop end values, the types must be compatible
 --> tests/errors/loop_01.py:5:20
  |
5 |     for i in range(15, i16(-1), i16(-1)):
  |                    ^^  ^^^^^^^ type mismatch ('i32' and 'i16')
